shell/browser/net/proxying_url_loader_factory.h

```diff
@@ -252,7 +253,7 @@ class ProxyingURLLoaderFactory
   bool ShouldIgnoreConnectionsLimit(const network::ResourceRequest& request);
 
   // Passed from api::WebRequest.
-  raw_ptr<WebRequestAPI> web_request_api_;
+  cppgc::Persistent<api::WebRequest> web_request_api_;
```

Comment:
should be `cppgc::WeakPersistent` to align with previous raw_ptr expectation

Reply:
fixed in 1b0cda95905aa22dfafbf0aeed7074ce46eb2821